src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,17 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in list.
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>(){
```

Comment:
You can also consider changing `Person` to `Comparable` or using Java 8 Streams (though this is fine too).